src/Fixer/Phpdoc/PhpdocTypesOrderFixer.php

```diff
@@ -189,7 +189,7 @@ private function sortTypes(TypeExpression $typeExpression): array
     {
         $normalizeType = static fn (string $type): string => Preg::replace('/^\(*\??\\\?/', '', $type);
 
-        $typeExpression->sortTypes(
+        $res = $typeExpression->sortTypes(
```

Comment:
Can we return from here or have a better name for `$res`?

Reply:
I prefer to assing the resulting `TypeExpression` to a variable first, the concept of `$res->getTypes()` was great when only union existed, but with intersect and complex types, the `$res->getTypes()` concept is only making the internals more complicated. But lets do not change this in this PR.

Reply:
> I prefer to assing the resulting `TypeExpression` to a variable first

I don't get it what is the benefit of it and what would change in behaviour if it would be `return $typeExpression->sortTypes(/* ... */)->getTypes();`.

Reply:
If you insist on this change I can land it, but I prefer to keep it as is for easier debugging/orientation.

Reply:
Let's change the `$res` to something meaningful, that's the way.

I don't insist or changing to chaining calls.

Reply:
`$resType` or suggest please.

Reply:
`$sortedTypeExpression` or simply `$typeExpression`?

Whenever I see `$res` I never know if it stands for `$result` or `$resource`.

Reply:
addressed

Reply:
@kubawerlos can this PR be merged now - I need it for another PR